internal/DateTime/DateTime.module.less

```diff
@@ -1,8 +1,29 @@
 // DateTime.module.less
 //
+@import "../../styles/skin.less";
+@import "../../styles/variables.less";
 
 .dateTime {
-	display: inline-block;
+	position: relative;
+	display: inline-flex;
+	align-items: center;
+	width: fit-content;
+
+	&::before {
```

Comment:
I'm not sure we should do this to each of DatePicker and TimePicker. But since we don't have specific UX, let's just revisit this later.